core-util-watcher/index.js

```diff
@@ -0,0 +1,62 @@
+#! /usr/local/bin/node --trace-warnings
+
+// For the package whose source code is at
+// https://github.com/ethereum-optimism/optimism/blob/develop/packages/core-utils/src/watcher.ts
+
+const Watcher = require('@eth-optimism/watcher').Watcher
+const JsonRpcProvider = require('@ethersproject/providers').JsonRpcProvider
+
+const infuraID = "<< PUT YOUR OWN VALUE HERE >>"
+const l1ToL2_L1TxHash = '0x65d50ed9d1edd7f20e97229667b5fc892fb23b128c8baca960c47c74fde9d4ed'
+const l2ToL1_L2TxHash = "0xa018b12df969c0f65aec3c3d08c94332525a309cdec38206eef74319c1d1dd30"
+
+const kovanWatcher = new Watcher({
+    l1: {
+      provider: new JsonRpcProvider(`https://kovan.infura.io/v3/${infuraID}`),
+      messengerAddress: '0x4361d0F75A0186C05f971c566dC6bEa5957483fD'
+    },
+    l2: {
+      provider: new JsonRpcProvider(`https://kovan.optimism.io`),
+      messengerAddress: '0x4200000000000000000000000000000000000007'
+    }
+  })
+
+
+  const mainnetWatcher = new Watcher({
+    l1: {
+      provider: new JsonRpcProvider(`https://mainnnet.infura.io/${infuraID}`),
+      // On Kovan, the value for mainnet is
```

Comment:
What do you mean by this comment "On Kovan, the value for mainnet is" ?

Reply:
That I forgot to remove the comment :-(

Reply:
I should have deleted that comment.